window/WindowAsAService/Pages/_Layout.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@using Microsoft.AspNetCore.Components.Web
+@namespace WindowAsAService.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WindowAsAService</title>
+    <base href="~/" />
+    <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
+    <link href="css/site.css" rel="stylesheet" />    
+    	<link href="@Url.Content("lib/blazor-ui/swatches/default-ocean-blue.css")" rel="stylesheet" type="text/css" />
+	<script src="_content/Telerik.UI.for.Blazor/js/telerik-blazor.js" defer></script>
```

Comment:
* The CSS file does not exist at that location.
* Using local assets is risky for projects that will go outdated and someone will update only the NuGet package in the future, without doing the same for the assets. Use static assets instead.
* Inconsistent tabbing.

Reply:
The CSS file is automatically generated by the VSX and is working as expected, the theme colors are okay, does it break for you?

We use static files, the configuration is present in the Program.cs (`app.UseStaticFiles();`). Do you mean something different?

We should report that tabbing to the VSX team as it is automatically generated, not sure if it is wrong, but does seem strange. Maybe it is done so because this is the "dynamic" file reference that you need to change if one wants a custom theme.

Reply:
I think it is better to use static asset in our sample projects to facilitate update. Moreover, we are insterested in the functionality for such projects and not the theme. The latest version of VSX templates have an option to choose cdn, or local resource. We do not use static assets to assure the presence of all swatches.

Reply:
* This file is still not committed to the repo: `lib/blazor-ui/swatches/default-ocean-blue.css`
* It is better to register the theme as a static asset (`_content/Telerik.UI.for.Blazor/css/kendo-theme-default/all.css`) to ensure seamless NuGet package version update in the future. Otherwise anyone will have to replace the CSS file manually.